packages/manager/modules/pci/src/projects/project/training/job.service.js

```diff
@@ -25,6 +26,26 @@ export default class PciProjectTrainingJobService {
       // remove shutdown before submitting as it is forbidden server side and irrelevant when resubmitting
       toSubmit = (({ shutdown, ...j }) => j)(jobSpec);
     }
+    if (jobSpec.volumes) {
+      // API does not accept volumes' internal property. We remove it before sending the request.
+      toSubmit.volumes = jobSpec.volumes.flatMap((volume) => {
+        let toRemove = false;
+        let internalKey;
+        Object.keys(volume).forEach((key) => {
+          if (volume[key] && volume[key].internal !== undefined) {
+            if (volume[key].internal === true) {
+              toRemove = true;
+            } else {
+              internalKey = key;
+            }
+          }
```

Comment:
**question:** Can only one `key` has `internal` property in the `volume` obj? If multiple keys can have internal property, this logic would only remove from the last key where internal is not set to true.

Reply:
Good point. As far as I know, we should not have multiple `internal` keys on volumes. However, I have changed the function to remove all of them, just in case: https://github.com/ovh/manager/pull/7423/commits/6f43fa3401e12a0af4f367ce703845f5136344c2